ci/prow/make_config.go

```diff
@@ -1309,6 +1313,14 @@ func main() {
 	if len(flag.Args()) != 1 {
 		log.Fatal("Pass the config file as parameter")
 	}
+
+	flakesreporterDockerImage = *dockerImagesBase + "/" + *flakesreporterDockerImageName
```

Comment:
nit, use path.Join instead of string concatenation since `*dockerImagesBase` could end with `/`

Reply:
Done.